### DISCUSSION TITLE
Xaml editor for vscode?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [picoe/Eto#2065] [Q&A] [answered]
Is there a way to edit eto forms and see the edit real time with xaml in vcode?

**Comment by cwensley** [comment, marked as answer]
Not currently, but I'd love to get this going as VS Code is my current editor of choice.  Not sure if/when I'll have time to do it though, there are some technical hurdles that would need to be overcome to integrate it into the editor.
